### ISSUE TITLE
[Accessibility] 'What's accessibility' link doesn't work

### ISSUE THREAD

**Issue by bruno-garcia** [getsentry/sentry#61246]
![image](https://github.com/getsentry/sentry/assets/1633368/c0d47628-6242-4f60-bff4-7f2ff9f0cf5e)

![image](https://github.com/getsentry/sentry/assets/1633368/7dc37178-a682-4b1c-8b93-c59daa7981e3)

**Comment by bruno-garcia** [comment]
@billyvg shouldn't this have raised Dead and Rage clicks?
In the reply I linked I clicked several times (on purpose to try out rage clicks) but it didn't create it

**Comment by bruno-garcia** [comment]
Ah I didn't link the reply, sorry
